lms/djangoapps/mailing/management/commands/mailchimp_sync_course.py

```diff
@@ -7,12 +7,10 @@
 import random
 from collections import namedtuple
 from itertools import chain
-from optparse import make_option
 
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
 from mailsnake import MailSnake
 from opaque_keys.edx.keys import CourseKey
-
 from student.models import UserProfile, unique_id_for_user
```

Comment:
Not that it matters too much one way or the other, but this blank line was probably here to visually separate the external package dependencies from the ones located within edx-platform.

Reply:
Yea, I see that now...do we want to go against the grain of isort?